docker/worker/Dockerfile

```diff
@@ -19,4 +21,6 @@ RUN apt-get install -y dnsutils
 
 COPY bin/worker /app/bin/worker
 
+USER engine
```

Comment:
Celery worker was complaining about running as root, and our manual installation docs say to create a user, so we mine as well do what a manual setup does as well!